app/util/conf.py

```diff
@@ -0,0 +1,38 @@
+import yaml
+from util.project_paths import JIRA_YML
+
+
+def read_yml_file(file):
+    with file.open(mode='r') as file:
+        return yaml.load(file, Loader=yaml.FullLoader)
+
+
+class JiraSettings:
+
+    def __init__(self):
+        obj = read_yml_file(JIRA_YML)
+        self.hostname = obj['settings']['env']['application_hostname']
+        self.protocol = obj['settings']['env']['application_protocol']
+        self.port = obj['settings']['env']['application_port']
+        self.postfix = obj['settings']['env']['application_postfix'] or ""
+        self.admin_login = obj['settings']['env']['admin_login']
+        self.admin_password = obj['settings']['env']['admin_password']
+        self.concurrency = obj['settings']['env']['concurrency']
+
+    def __repr__(self):
```

Comment:
why do we need this method?

Reply:
It's like a best practice. It can be also useful for debugging.